Tensile/KernelWriter.py

```diff
@@ -2614,28 +2680,28 @@ def initKernel(self, kernel, tensorParametersA, tensorParametersB ):
     self.numReadVectorComponentsA = kernel["GlobalLoadVectorWidthA"] \
         if (self.readTileDimComponentsA \
         or self.readUnrollDimComponentsA) else 1
-    self.numWriteVectorComponentsA = kernel["GlobalLoadVectorWidthA"] \
-        if (self.writeTileDimComponentsA \
-        or self.writeUnrollDimComponentsA) else 1
-    self.numReadTileVectorComponentsA = kernel["GlobalLoadVectorWidthA"] \
-        if self.readTileDimComponentsA else 1 # for branches
+    # self.numWriteVectorComponentsA = kernel["GlobalLoadVectorWidthA"] \
```

Comment:
We can probably just delete all this unused code unless its going to be needed again soon.